changes/4406.added

```diff
@@ -0,0 +1 @@
+Added `role` field to `Interface` and `VMInterface` models
```

Comment:
```suggestion
Added `role` field to `Interface` and `VMInterface` models.
```

Reply:
Let's go ahead and add this as a feature in the 2.3 release note release overview as well (now that such a file exists!)